config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java

```diff
@@ -92,13 +103,22 @@ public FileConfiguration(String name, boolean allowDynamicRefresh) {
             targetFilePath = name.substring(SYS_FILE_RESOURCE_PREFIX.length());
```

Comment:
targetFileLastModified is null
tempLastModified > targetFileLastModified will be NPE

Reply:
When allowDynamicRefresh is  false,the code won't be executed.

Reply:
When config.type is file and file.name is "file: ..."

Reply:
OK.

Reply:
Here and ConfigurationFactory have been iterated many times, it is difficult to read now, hope can be refactored.

Reply:
NPE fixed